--- Doxyfile	2022-03-20 23:17:24.113014800 +0000
+++ ../patch/Doxyfile	2022-03-20 23:17:50.692793200 +0000
@@ -829,7 +829,7 @@ WARN_LOGFILE           =
 # spaces. See also FILE_PATTERNS and EXTENSION_MAPPING
 # Note: If this tag is empty the current directory is searched.
 
-INPUT                  =
+INPUT                  = ../src/cpp_impl/
 
 # This tag can be used to specify the character encoding of the source files
 # that doxygen parses. Internally doxygen uses the UTF-8 encoding. Doxygen uses
