--- ../docs/Doxyfile	2022-04-02 23:49:47.413597700 +0000
+++ Doxyfile	2022-04-02 23:47:56.579284100 +0000
@@ -829,7 +829,8 @@ WARN_LOGFILE           =
 # spaces. See also FILE_PATTERNS and EXTENSION_MAPPING
 # Note: If this tag is empty the current directory is searched.
 
-INPUT                  =
+INPUT                  = ../src/cpp_impl \
+                         ../src/cpp_impl_boost
 
 # This tag can be used to specify the character encoding of the source files
 # that doxygen parses. Internally doxygen uses the UTF-8 encoding. Doxygen uses
